depends/packages/qt.mk

```diff
@@ -248,13 +248,13 @@ define $(package)_preprocess_cmds
   patch -p1 -i $($(package)_patch_dir)/fix_android_jni_static.patch && \
   patch -p1 -i $($(package)_patch_dir)/no-xlib.patch && \
   patch -p1 -i $($(package)_patch_dir)/dont_hardcode_x86_64.patch && \
-  patch -p1 -i $($(package)_patch_dir)/fix_limits_header.patch && \
```

Comment:
patch is removed, but file is still here

Reply:
Resolved in latest push